src/esw/brushless.py

```diff
@@ -108,77 +118,91 @@ async def _send_command(self) -> None:
         command = self._command
         self.command_lock.release()
```

Comment:
lock is released with no accompanying  acquire?

Reply:
Its in the line right above

Reply:
I'll probably change all to use guards instead tho with the with syntax

Reply:
my bad, github was collapsing those lines by default not sure why